src/plugins/discover/public/application/view_components/canvas/discover_canvas.scss

```diff
@@ -12,6 +12,12 @@
     position: relative;
   }
 
+  &_actions {
+    position: absolute;
+    top: 0;
+    right: 30px;
```

Comment:
How did we land at 30px? We should use $euiSizeL or XL.

Reply:
Changed it to $euiSizeL, thanks!